pkg/amazonlinux2016.09/Dockerfile

```diff
@@ -180,11 +181,14 @@ CMD [ "if [ -f /data/hubble_buildinfo ] ; then echo \"\" >> /hubble_build/hubble
        -n hubblestack \
        -v ${HUBBLE_VERSION} \
        --iteration ${HUBBLE_ITERATION} \
-       --config-files /etc/osquery/osquery.conf \
+       --url ${HUBBLE_URL} \
+       --description \"${HUBBLE_DESCRIPTION}\" \
+       --rpm-summary \"${HUBBLE_SUMMARY}\" \
+       --config-files /etc/hubble/hubble \
```

Comment:
This used to be here but I explicitly removed it. The idea is that we have our users put their config in `/etc/hubble/hubble.d/hubble.conf` so that we can update `/etc/hubble/hubble` properly. If you add it as a `--config-files` entry, then if the user touches it, yum will stop updating it.

Can you please remove this from all of the dockerfiles?

Reply:
Sure, didn't know that. Fixed.